searx/templates/oscar/search.html

```diff
@@ -3,7 +3,7 @@
   <div class="row">
     <div class="col-xs-12 col-md-8">
       <div class="input-group search-margin">
-        <input type="search" name="q" class="form-control" id="q" placeholder="{{ _('Search for...') }}" aria-label="{{ _('Search for...') }}" autocomplete="off" value="{{ q }}">
```

Comment:
In https://github.com/asciimoo/searx/blob/master/searx/static/plugins/js/vim_hotkeys.js the key is ```i``` (without control keys). Not sure if it creates confusion or not.

It seems there is no de facto standard : 
* ddg uses ```/``` and ```h```
* wikipedia uses ```h```
* github uses ```/``` and ```s```

Reply:
> Not sure if it creates confusion or not 

both work .. why should one be confused?

> It seems there is no de facto standard ...

As far I can tell, they do not use attribute ````accesskey```` (it is without control keys)

Reply:
From user point of view, I have to know that it is neither ```<alt>``` + ```<shift>``` + ```i``` nor ```s``` but the other way around.